requirements/django.txt

```diff
@@ -6,28 +6,26 @@
 #
 appdirs==1.4.3
 backports.os==0.1.1
-boto3==1.10.17            # via fs-s3fs
-botocore==1.13.17         # via boto3, s3transfer
+boto3==1.10.18            # via fs-s3fs
+botocore==1.13.18         # via boto3, s3transfer
 django-pyfs==2.0
 django==1.11.26
-docutils==0.15.2          # via botocore
```

Comment:
All of the dependencies of `botocore` vanished; try adding `--rebuild` to the pip-compile commands in `make upgrade` to see if that fixes it.